lib/teiserver_web/router.ex

```diff
@@ -561,6 +561,8 @@ defmodule TeiserverWeb.Router do
 
     # User stuff
     put("/users/gdpr_clean/:id", UserController, :gdpr_clean)
+    get("/users/delete_user/:id", UserController, :delete_user)
+    put("/users/delete_user/:id", UserController, :delete_user)
```

Comment:
The routing is roughly REST-ish, so it would be better to use the `delete` function (and associated http verb) to delete the user.

Reply:
should be correct now